lib/report_formatter/timeline_message.rb

```diff
@@ -34,6 +34,40 @@ def dest_host_name
       "<a href=/host/show/#{to_cid(@event.dest_host_id)}>#{text}</a>" if @event.dest_host_id
     end
 
+    def target_name
+      if @event.target_name # Create the title using Policy description
+        e_title = @event.target_name
+      elsif @event.miq_policy_id && MiqPolicy.exists?(@event.miq_policy_id) # or Policy name
+        e_title = MiqPolicy.find(@event.miq_policy_id).name
+      else
+        e_title = "Policy no longer exists"
```

Comment:
Will the `"Policy no longer exists"` string be propagated into UI? If so, it should be inside `_()`.